### DISCUSSION TITLE
Is it possible to update the footer navigation to use the parent's title?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [vuejs/vitepress#2112] [Q&A] [answered]
Is there a way to change the footer navigation to use the Parent title if it is a child?
![CleanShot 2023-03-21 at 00 46 16](https://user-images.githubusercontent.com/1639302/226444710-ae7ed7ce-864f-40fd-8c31-26cb849ed05f.png)

**Comment by brc-dd** [comment, marked as answer]
https://vitepress.dev/reference/default-theme-prev-next-links#prev-next-links
